kalite/store/models.py

```diff
@@ -51,7 +51,7 @@ class StoreTransactionLog(DeferredCountSyncedModel):
     Model to track student 'expenditure' of points.
     """
 
-    # TODO-BLOCKER(rtibbles): Update this to "0.13.0" (or whatever the release version number is at the time this goes upstream)
+    # TODO(rtibbles): Update this to "0.13.0" (or whatever the release version number is at the time this goes upstream)
```

Comment:
The store app isn't activated on the distributed server in 0.13.0, but it's activated on the central server still, since otherwise nalanda would lose all the data they have (and are actively using) in there. So, we need to keep the models in there for now.